crates/core_types/src/link/mod.rs

```diff
@@ -84,7 +83,6 @@ impl<S: Into<String>> LinkMatch<S> {
         let re_string: String = match self {
             LinkMatch::Any => ".*".into(),
             LinkMatch::Exactly(s) => "^".to_owned() + &regex::escape(&s.into()) + "$",
```

Comment:
do we still even need this function? - this is a pretty inefficient way to match a full string (similarly for the Any above).

Reply:
Yeah we don't need this anymore, wasnt using it anywhere